src/patternfly/components/Expandable/docs/code.md

```diff
@@ -0,0 +1,21 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-expandable__toggle` | Indicated that the expandable content is visible. **Required** |
+| `hidden` | `.pf-c-expandable__content` | Indicates that the expandable content is hidden. **Required**. |
+| `aria-hidden="true"` | `.pf-c-expandable__toggle-icon` | Hides from screen readers. |
+| `aria-controls="[id of element controlled]"` | `.pf-c-expandable__toggle-icon` | Identifies the section controlled by the toggle button. |
```

Comment:
I would also add an `id` line that applies to `.pf-c-expandable__content` since that's needed to work with `aria-controls`